runtime/Stdlib_Iterator.res

```diff
@@ -7,17 +7,38 @@ type value<'a> = {
 }
 
 @send external next: t<'a> => value<'a> = "next"
-external toArray: t<'a> => array<'a> = "Array.from"
+@send
+external toArray: t<'a> => array<'a> = "toArray"
 external toArrayWithMapper: (t<'a>, 'a => 'b) => array<'b> = "Array.from"
 
-let forEach = (iterator, f) => {
```

Comment:
I'm for this change :+1:

`Iterator.forEach`'s behaviour was different to `Iterator.prototype.forEach` as the latter only calls its callback with yielded values, [not the iterator's final return value](https://github.com/es-shims/iterator-helpers/blob/06515e7d1f5d5e7c98777eba6aec50d1cdaf6a4d/Iterator.prototype.forEach/implementation.js#L34-L36).